tox.ini

```diff
@@ -35,6 +35,7 @@ deps=
   hypothesis[datetime]
   selenium==3.0.2
   pymongo==3.4.0
+  toml
```

Comment:
If we agree to a version requirement in `setup.py`, it'll need to be reflected here.